.github/workflows/print_github_metrics.py

```
@@ -0,0 +1,306 @@
+        
+        # Runner info
+        runner_labels = current_job.get('labels', [])
+        metrics["runner_labels"] = runner_labels
+        metrics["runner_name"] = current_job.get('runner_name', 'unknown')
+        
+        # Duration and queue time
+        if created_at and completed_at:
+            try:
+                duration = (pd.to_datetime(completed_at) - pd.to_datetime(created_at)).total_seconds()
+                metrics["job_duration_seconds"] = duration
+            except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)